signer/src/storage/postgres.rs

```diff
@@ -1531,10 +1531,12 @@ impl super::DbRead for PgStore {
 
     /// Returns the number of rows in the `dkg_shares` table.
```

Comment:
```suggestion
    /// Returns the number of unverified rows in the `dkg_shares` table.
```

Reply:
Done in https://github.com/stacks-network/sbtc/pull/1317/commits/660698878265dffdc7768171691efb4bee38f040